packages/astro-font/utils.ts

```diff
@@ -255,7 +261,7 @@ async function getFallbackFont(fontCollection: Config): Promise<Record> {
           if (res) {
             fonts.push({
               style: i.style,
-              weight: i.weight,
+              weight: i.weight?.toString(),
```

Comment:
this might be dumb to ask but is 200.toString() === '200'.toString()?

In the browser console, 200.toString() seems to be throwing an error

Reply:
doing it with a variable is different
![image](https://github.com/rishi-raj-jain/astro-font/assets/69633530/3103825e-d0e6-431c-9688-48de469a57b9)